lib/components/template/layer/LayerManager.vue

```diff
@@ -124,6 +129,11 @@ v-layout(column)
         this.showNewLayerDialog = false
       },
 
+      copyLayer(layer) {
+        this.copyTemplateLayer({ template: this.template, layer })
+        this.showNewLayerDialog = false
```

Comment:
Is this necessary? That dialog isn't open is it?

Reply:
Totally, copy-pasting issue, XD.

Reply:
Fixed it.